ui/pages/confirmations/hooks/alerts/useBlockaidAlerts.ts

```diff
@@ -82,11 +125,22 @@ const useBlockaidAlerts = (): Alert[] => {
       return [];
     }
 
-    return [normalizeProviderAlert(securityAlertResponse, t)];
+    let reportUrl = ZENDESK_URLS.SUPPORT_URL;
+    if (stringifiedJSONData) {
+      const encodedData =
+        zlib?.gzipSync?.(stringifiedJSONData) ?? stringifiedJSONData;
+
+      reportUrl = `${FALSE_POSITIVE_REPORT_BASE_URL}?data=${encodeURIComponent(
+        encodedData.toString('base64'),
+      )}&utm_source=${SECURITY_PROVIDER_UTM_SOURCE}`;
+    }
+
+    return [normalizeProviderAlert(securityAlertResponse, t, reportUrl)];
```

Comment:
is this duplicating existing logic that is already in blackaid-banner-alert.js?

Reply:
`blackaid-banner-alert` is old code that we will eventually get rid of and replace by this code.